lib/url-pattern/url-match-pattern.js

```diff
@@ -103,11 +123,13 @@ _.assign(UrlMatchPattern.prototype, /** @lends UrlMatchPattern.prototype */ {
     testProtocol: function (protocol) {
         var matchRegexObject = this._matchPatternObject;
 
-        // Do not match if protocol is invalid
+        if (this._allowProtocols) {
+            return _.indexOf(this.protocols, protocol) !== -1;
```

Comment:
will use _.includes since _.contains is not there in `lodash` but in underscore.